setup.py

```diff
@@ -39,11 +40,18 @@
     test_requirements = [line for line in requirements_file.read().splitlines()
                          if not line.startswith('#')]
 
-
 extras_require = {
     'standard': ['netifaces', 'numpy', 'dpkt'],
     'async': ['curio>=1.2', 'trio>=0.12.1'],
 }
+
+if sys.version_info == (3, 6):
+    # For PVA support of dataclasses
+    extras_require['standard'].append('dataclasses')
+
+# TODO this is hard requirement at the moment:
+extras_require['standard'].append('parsimonious')
```

Comment:
Is all the CA functionality still importable usable without this? I'm OK making this a hard requirement for PVA for now.

Reply:
Since we don't import `pva` at the top-level, it should be. Should we make this into a separate extras category?

Reply:
I'm 50/50 on that. Whatever you think is best.